networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2P2PNetwork.java

```diff
@@ -179,11 +179,21 @@ public void onSyncStateChanged(final boolean isInSync, final boolean isOptimisti
     gossipForkManager.onOptimisticHeadChanged(isOptimistic);
   }
 
-  private boolean isCloseToInSync() {
+  @VisibleForTesting
+  boolean isCloseToInSync() {
+    final Optional<UInt64> currentEpoch = recentChainData.getCurrentEpoch();
+    if (currentEpoch.isEmpty()) {
+      return false;
+    }
+
+    final int maxLookaheadEpochs = spec.getSpecConfig(currentEpoch.get()).getMaxSeedLookahead();
+    final int maxLookaheadSlots = spec.slotsPerEpoch(currentEpoch.get());
+    final int distance = maxLookaheadSlots * maxLookaheadEpochs;
```

Comment:
this distance variable is what i was referring to that should be called `maxLookaheadSlots`
and probably the previous either can not exist, or should be something like `slotsPerEpoch`